e2e/e2e.go

```diff
@@ -127,15 +131,19 @@ func CreateSysTestSuite(t *testing.T) *SysTestSuite {
 	appCfg := DefaultTestConfig()
 
 	slackServer := NewTestSlackServer()
-	appCfg.AlertConfig.SlackConfig.URL = slackServer.Server.URL
 
 	pagerdutyServer := NewTestPagerDutyServer()
-	appCfg.AlertConfig.MediumPagerDutyCfg.AlertEventsURL = pagerdutyServer.Server.URL
+
+	appCfg.AlertConfig.RoutingCfgPath = "alert-routing-cfg.yaml"
+	appCfg.AlertConfig.PagerdutyAlertEventsURL = pagerdutyServer.Server.URL
 
 	pess, kill, err := app.NewPessimismApp(ctx, appCfg)
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	UpdateSlackUrls(appCfg.AlertConfig, slackServer.Server.URL)
```

Comment:
Let's deprecate this in favor of using preset ports for our testing servers

Reply:
[19ad3ed](https://github.com/base-org/pessimism/pull/149/commits/19ad3edcaa362b6d79e24558c26bdfdcdbb2658e)